src/main/scala/mesosphere/marathon/core/pod/Volume.scala

```diff
@@ -0,0 +1,17 @@
+package mesosphere.marathon.core.pod
+
+sealed trait Volume extends Product with Serializable {
```

Comment:
Just curious: why Product and Serializable? (case class implements that, I know)